application/config/client_settings_test.go

```diff
@@ -29,11 +29,11 @@ func TestGetEnabledProducts_DefaultValues(t *testing.T) {
 	t.Setenv(ActivateSnykIacKey, "set it to anything to make sure it is reset")
 	t.Setenv(ActivateSnykContainerKey, "set it to anything to make sure it is reset")
 	t.Setenv(ActivateSnykAdvisorKey, "set it to anything to make sure it is reset")
-	os.Unsetenv(ActivateSnykOssKey)
-	os.Unsetenv(ActivateSnykCodeKey)
-	os.Unsetenv(ActivateSnykIacKey)
-	os.Unsetenv(ActivateSnykContainerKey)
-	os.Unsetenv(ActivateSnykAdvisorKey)
+	_ = os.Unsetenv(ActivateSnykOssKey)
+	_ = os.Unsetenv(ActivateSnykCodeKey)
+	_ = os.Unsetenv(ActivateSnykIacKey)
+	_ = os.Unsetenv(ActivateSnykContainerKey)
+	_ = os.Unsetenv(ActivateSnykAdvisorKey)
```

Comment:
Why do we do this? Can't we use `t.Setenv(key, "")`?

Reply:
Empty string is not the same as no variable. I think that's why.

Reply:
I tried locally and the test passed.

I think the only difference between an empty env-var and an unset one is that `os.LookupEnv("EMPTY_VAR_KEY")` will return a 2nd output value that's a boolean specifying that the env-var exists, and this is only used in `Config.loadFile`. So basically if you have an empty env var it would not be overwritten by the value in the config file.

I think it's better if we don't differentiate between an empty env-var and a missing one.

I don't think that we have to address it in this PR though